var/spack/repos/builtin/packages/py-tensorflow-hub/package.py

```diff
@@ -49,8 +49,6 @@ def install(self, spec, prefix):
             "--jobs={0}".format(make_jobs),
             # Enable verbose output for failures
             "--verbose_failures",
-            # Show (formatted) subcommands being executed
-            "--subcommands=pretty_print",
             "--spawn_strategy=local",
             # Ask bazel to explain what it's up to
             # Needs a filename as argument
```

Comment:
Did you want to remove these too for consistency? Same in other files.

Reply:
Yeah, I think it's generating about 250MB of output, but didn't verify this...